mk/environ.mk

```diff
@@ -7,6 +7,12 @@ SYSTYPE			:=	$(shell uname)
 GIT_VERSION := $(shell git rev-parse HEAD | cut -c1-8)
 EXTRAFLAGS += -DGIT_VERSION="\"$(GIT_VERSION)\""
 
+# Add missing parts from libc and libstdc++ for all boards
+EXTRAFLAGS += -I$(SKETCHBOOK)/libraries/AP_Common/missing
+
+# Hardcoded libraries/AP_Common/missing/cmath defines in "make" to retain the current behavior
+EXTRAFLAGS += -DHAVE_CMATH_ISFINITE='1'
```

Comment:
> What about the others?

This is for the make build, our goal here is to retain the current behavior, we'll need only std::isfinite, the other ones we use can as a macro.

> Also, ='1' isn't really necessary.

It is necessary in order to `using std::isfinite;` for the case I mentioned above.